src/main/java/WayofTime/bloodmagic/ritual/types/RitualPortal.java

```diff
@@ -172,7 +170,7 @@ public void stopRitual(IMasterRitualStone masterRitualStone, BreakType breakType
             }
         }
 
-        portalRitualTag.removeTag(PORTAL_ID_TAG);
+        portalRitualTag.removeTag(portalRitualTag.getString(PORTAL_ID_TAG));
```

Comment:
I keep looking at this and I can't help but feel it's wrong. Does `removeTag` not take the key?

Reply:
It does not. That was what probably was the most annoying to track down, because it just wouldn't remove previous portal locations.